pennylane/beta/queuing/queuing.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the :class:`QueuingContext` abstract base class.
+"""
```

Comment:
This file is, for all intents and purposes, identical to the existing `pennylane._queuing` file. The only difference is the `active_context` method -- it returns only the queue on the top of the stack, not all queues on the stack. This is required to support nesting.

Reply:
:+1:

Is this why there aren't any new tests written for this file? i.e., we can use `test_queueing.py` when taking this out of beta?

Reply:
I will duplicate the test file as well!

Reply:
done!